client/dogs/FormPage.js

```diff
@@ -243,82 +269,111 @@ export default class IndexPage extends React.Component {
             </div>
 
             <div className="txt">
-              <label htmlFor="lookup-dogAge" className="txt-l">
+              <label htmlFor="lookup-dogYear" className="txt-l">
                 Dog Age (Years)
               </label>
               <input
-                id="lookup-dogAge"
-                name="dogAge"
-                value={this.state.dogAge}
+                id="lookup-dogYear"
+                type="text"
+                name="dogYear"
+                value={this.state.ageYears}
                 onChange={this.handleInputChange}
                 placeholder="2, 0.5, 17..."
                 className="txt-f"
               />
             </div>
 
-            <div className="sel">
-              <label htmlFor="lookup-breed" className="sel-l">
-                Dog Breed
+            <div className="txt">
+              <label htmlFor="lookup-dogMo" className="txt-l">
+                Dog Age (Months)
               </label>
-              <div className="sel-c">
-                <select
-                  name="breed"
-                  id="lookup-breed"
-                  value={this.state.breed}
-                  className="sel-f">
-                  <option value="Alaskan Malamute">Alaskan Malamute</option>
-                  <option value="Alaskan Husky">Alaskan Husky</option>
-                  <option value="Pit Bull Terrier">Pit Bull Terrier</option>
-                  <option value="Bulldog">Bulldog</option>
-                  <option value="Australian Shephard">
-                    Australian Shephard
-                  </option>
-                  <option value="Basset Hound">Basset Hound</option>
-                  <option value="Italian Greyhound">Italian Greyhound</option>
-                  <option value="Otterhound">Otterhound</option>
-                  <option value="Pharoah Hound">Pharoah Hound</option>
-                  <option value="Plott Hound">Plott Hound</option>
-                  <option value="Portuguese Water Dog">
-                    Portuguese Water Dog
-                  </option>
-                  <option value="Whippet">Whippet</option>
-                </select>
+              <input
+                id="lookup-dogMo"
+                type="text"
+                name="dogMo"
```

Comment:
you'll need to update the names of these fields to match the properties in `state`